Source/Provers/SMTLib/SMTLibProcess.cs

```diff
@@ -65,13 +66,22 @@ public SMTLibProcess(SMTLibOptions libOptions, SMTLibProverOptions options)
         toProver = prover.StandardInput;
         prover.BeginErrorReadLine();
         prover.BeginOutputReadLine();
+        var _ = ReadResponses();
```

Comment:
But the output of readresponses is not awaited, right?

Reply:
ReadResponses itself is awaiting tasks, that store a references to it.

Reply:
> If you await a task, like ReadResponses, then it might still be garbage collected because you're not storing a reference to it.

How do you read its output after it completes if you don't have (under the hood) a reference to it?

Reply:
I've changed this code so the discussion is moot, but ReadResponses() was writing to a queue from which was read.